design/src/App.scss

```diff
@@ -77,6 +78,115 @@ $hover-color: #f8cc80;
     }
 }
 
+.glogg-h1 {
+    font-family: 'Merienda', cursive;
+    font-size: 35pt;
+    color: #ffffff;
+    text-align: center;
+}
+
+.glogg-row1{
+    padding: 50px;
+    background-color: #9D8471;
+}
+
+.glogg-ribbon-text{
+    font-family: 'Merienda', cursive;
+    font-size: 18pt;
+    color: #ffffff;
+    text-align: center;
+    position: absolute;
+    width: 8em;
+    height: 10em;
+    z-index: 9999;
+    top: 0;
+    right: 10%;
+    background: $ribbon-color;
+    padding: 5px 0;
+}
+
+.glogg-ribbon{
+    z-index: 99999;
+    position: relative;
+    margin: 0 5px;
+}
+
+.glogg-ribbon-text:before {
```

Comment:
The `:before` and `:after` selectors could be grouped under row 108, i.e. under `.glogg-ribbon` like this, to utilize Sass to the max:

```
.glogg-ribbon {
  z-index: 999;
  // ...
  &:before {
    // code here
  }
}
```

Reply:
yes, done